### ISSUE TITLE
Dynamic filtering names to be sorted

### ISSUE THREAD

**Issue by jetienne** [avo-hq/avo#1898]
### Feature
Dynamic filters are currently unsorted. For better clarity, it would be beneficial to sort them in ascending order by name.

### Screenshots or screen recordings
![Screenshot 2023-08-11 at 13 23 03](https://github.com/avo-hq/avo/assets/201689/35a75b47-3182-4c59-aa26-c65d57a79b1d)

**Comment by adrianthedev** [comment]
Makes sense! I think the order now is the one declared in the `fields` method. 
I added it to the near roadmap.
Thanks!
